democracy/tests/test_images.py

```diff
@@ -52,3 +53,52 @@ def test_38_get_hearing_check_section_with_images(api_client, default_hearing):
     assert 'sections' in data
     first_section = data['sections'][0]
     check_entity_images(first_section)
+
+
+@pytest.mark.parametrize('client, expected', [
+    ('api_client', False),
+    ('jane_doe_api_client', False),
+    ('admin_api_client', True)
+])
+@pytest.mark.django_db
+def test_unpublished_hearing_images_excluded(client, expected, request, default_hearing):
+    api_client = request.getfuncargvalue(client)
+
+    image = default_hearing.images.get(title=IMAGES['ORIGINAL'])
+    image.published = False
+    image.save(update_fields=('published',))
+
+    # /v1/hearing/ images field
+    data_1 = get_data_from_response(api_client.get(list_endpoint))[0]['images']
+
+    # /v1/hearing/<id>/ images field
+    data_2 = get_data_from_response(api_client.get(get_hearing_detail_url(default_hearing.id)))['images']
+
+    # /v1/hearing/<id>/images/
+    data_3 = get_data_from_response(api_client.get(get_hearing_detail_url(default_hearing.id, 'images')))
+
+    for data in (data_1, data_2, data_3):
```

Comment:
yeah true, sometimes I tend to be a bit lazy in tests.. changing and merging after that